bin/sandbox

```diff
@@ -85,11 +84,12 @@ unbundled bundle install --gemfile Gemfile
 
 unbundled bundle exec rake db:drop db:create
 
+# Still request "devise". Solidus will skip installing it but include the seeds
 unbundled bundle exec rails generate solidus:install \
   --auto-accept \
   --user_class=Spree::User \
   --enforce_available_locales=true \
-  --with-authentication=true \
+  --authentication="devise" \
```

Comment:
@RyanofWoods I'm able to run `bin/sandbox` with the original argument `--with-authentication=true`. Can you confirm?

Reply:
@RyanofWoods Oh wait, I didn't notice the deprecation warning in the `bin/sandbox` output:

```
     warning  Using `solidus:install --with-authentication` is now deprecated. Please use `--authentication` instead (see --help for the full list of options).
```

So, the `--authentication="devise"` argument is actually better :)

However, I'm not sure I understand if it's related to this comment in the commit message: "A flag will still be given to the Solidus generator so that includes solidus_auth_devise's seeds. It will skip installing it twice however." Can you clarify?

Reply:
The latter is from solidus master, --with-authentication is the legacy one (valid up until 3.2)

Reply:
@gsmendoza We still provide the authentication flag so that it appends the devise extension seeds to the sandbox, which is quite useful as it makes the default admin account. The Solidus installer is smart enough to skip adding the extension to the Gemfile though:
https://github.com/solidusio/solidus/blob/efe2b3a20d0577413e4810c008ea564e637c36a3/core/lib/generators/solidus/install/app_templates/authentication/devise.rb

I am more than happy to improve the commit msg if you wish 🙂

Reply:
@RyanofWoods Oh wait, I understand the commit message now. You're saying that we still need to call `solidus:install` with the authentication flag even if `solidus_auth_devise` has been added to the Gemfile. I got confused because I thought it was connected to the change from the deprecated authentication flag to the new one.

Reply:
Yes. But technically, it is not needed - it's just so a admin user gets created, which is very convenient.

Sorry for the confusion. It was then a good idea to update the commit msg 👍 I also moved the flag changing to another commit, because it was unrelated.